packages/worker/src/worker.ts

```diff
@@ -120,29 +142,19 @@ export interface WorkerConstructor {
 
 export class NativeWorker implements NativeWorkerLike {
   protected readonly native: native.Worker;
+  protected readonly pollFn: (worker: native.Worker, queueName: string) => Promise<ArrayBuffer>;
 
   public constructor() {
     this.native = native.newWorker();
+    this.pollFn = promisify(native.workerPoll) as any; // Typescript isn't smart enough to deduce the type here
```

Comment:
Can you reify it to a more specific type than `any` ?

Reply:
Ended up changing the callback type, apparently the types of callbacks in the nodejs apis do not have an optional result argument.